### PR TITLE
Resolve Wrong Css at "Minimum Qty Allowed in Shopping Cart" on Google Chrome issue25647

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

1. magento/magento2#25647: Resolve Wrong Css at "Minimum Qty Allowed in Shopping Cart" on Google Chrome issue25647

### Fixed Issues (if relevant)

1. magento/magento2#25647: Resolve Wrong Css at "Minimum Qty Allowed in Shopping Cart" on Google Chrome issue25647

### Manual testing scenarios (*)

1. Go to backend
2. Store->Configurations, Catalog->Inventory
3. Look at "Product Stock Options", "Minimum Qty Allowed in Shopping Cart"

image

#### Expected result 

1. Gray color as **Disabled**

image

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by edenduong** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rodrigowebjump** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by rodrigowebjump** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @rodrigowebjump, here is your [Magento instance](https://i-25648-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25648-2-3-develop.instances.magento-community.engineering/admin](https://i-25648-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by rodrigowebjump** [comment]
Hi @edenduong 

In my test, the quantity is not displayed as disabled. Please could you check this too?

image

**Comment by edenduong** [comment]
Hi @rodrigowebjump ,

It is ok now. Please check it. Thanks!

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by ptylek** [comment]
@magento give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [Magento instance](https://i-25648-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-25648-2-3-develop.instances.magento-community.engineering/admin](https://i-25648-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

File: app/design/adminhtml/Magento/backend/Magento_Config/web/css/source/_module.less [hunk 1]
```diff
@@ -147,4 +147,9 @@
             display: none;
         }
     }
+
+    select:disabled,
+    input:disabled {
+        .__form-control-pattern__disabled();
+    }
```
Comment (ptylek) [inline app/design/adminhtml/Magento/backend/Magento_Config/web/css/source/_module.less]: These changes apply also over all other input and select elements in Admin Panel, even those which already have disabled/[disabled]/.disabled styling applied.

**Review by ptylek — CHANGES_REQUESTED** [review]
Hi @edenduong! Thank you for your contribution!
I've seen those changes and I think that adding `admin__control-text` class to that input and `admin__control-select` would be a better idea instead of applying the same styles once again.
That's how it would look with those proper classes added:

image

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

File: app/design/adminhtml/Magento/backend/Magento_Config/web/css/source/_module.less (hunk 1 above)
Comment (edenduong) [reply app/design/adminhtml/Magento/backend/Magento_Config/web/css/source/_module.less]: Hi @ptylek :
I changed it to .admin__control-table-wrapper select:disabled and .admin__control-table-wrapper input:disabled. Please check it again. Thanks!

**Comment by edenduong** [comment]
Hi @ptylek : 

I have changed the source code. Please check it again. Thanks!

**Comment by ptylek** [comment]
@edenduong could you remove those css styles and try to add admin__control-text class to that input and admin__control-select to that select?

**Comment by ptylek** [comment]
@edenduong could you remove those css styles and try to add `admin__control-text` class to that input and `admin__control-select` to that select?

**Comment by edenduong** [comment]
> @edenduong could you remove those css styles and try to add `admin__control-text` class to that input and `admin__control-select` to that select?

Hi @ptylek, I tried it but it doesn't work. Because seems the input doesn't have parent class admin_control-text or select.

**Comment by edenduong** [comment]
Hi @ptylek. Ok, I understood. I will try again. In last comment, I missunderstand. Please wait me. Thank you very much!

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by edenduong** [comment]
Hi @ptylek,

It is ok now. Please check it again. Thanks!

**Comment by ptylek** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @ptylek. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @ptylek, here is your [new Magento instance](https://pr-25648.instances.magento-community.engineering).
Admin access: [https://pr-25648.instances.magento-community.engineering/admin](https://pr-25648.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Review by ptylek — APPROVED** [review]

**Comment by ptylek** [comment]
@edenduong Thanks for your contribution. I approved these changes.

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-6321](https://jira.corp.magento.com/browse/ENGCOM-6321) has been created to process this Pull Request
:eight_spoked_asterisk: @ptylek, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Bravo** [comment]
Hello @edenduong. Thanks for Your contribution. Here is a screenshot from a fresh install of the latest develop 2.3
MinQtyAll
Just as the Expected Result in the issue Your PR is going to fix. Could You please recheck it. Thank You.

**Comment by edenduong** [comment]
Hi @engcom-Bravo :

In 2.3-develop:

image

You can see All Group is white color background , not gray. It is disabled and should be Gray color like that:

image

Please check again. Thanks !

**Comment by engcom-Bravo** [comment]
Here You are @edenduong 
freshinstall
I'll cloned my code an our ago from the engcom repository.

**Comment by edenduong** [comment]
@engcom-Bravo: I think the browser is different so in my screen, the style is different a bit. But you can see it your video, the style of "All group" must be the same like that:

image

Now it is different. You can see.

Thanks!

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**